src/screens/Calculator/styles.scss

```diff
@@ -0,0 +1,43 @@
+@import './../../shared/main-styles.scss';
+
+.card {
+  border: 2px solid gray;
+  display: flex;
+  flex-direction: column;
+  width: 400px;
+  height: 450px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 100px;
+  box-shadow: 1px 4px 4px 1px gray;
+  border-radius: 8px;
+  background-color:$bg-card-color;
+}
+
+.title {
+  text-align: center;
+  letter-spacing: 2px;
+  color: $title-text-color;
+  border-radius: 8px;
+  font-size: 20px;
+  margin-bottom: 20px;
+  background: rgb(136,136,136);
+  background: $bg-title-color;
+}
+
+main {
```

Comment:
You should not use html tags as styles selectors. Create separate class.

Reply:
Done!